src/Sensors.cpp

```diff
@@ -99,31 +99,46 @@ void Sensors::setSampleTime(int seconds) {
 }
 
 /// set CO2 recalibration PPM value (400 to 2000)
-void Sensors::setCO2RecalibrationFactor(int ppmValue) {
-    if (getPmDeviceSelected().equals("SCD30")) {
+void Sensors::setCO2RecalibrationFactor(int ppmValue)
+{
+    Serial.print("DEVICE SELECTED: ");
+    Serial.println(getPmDeviceSelected());
+    if (getPmDeviceSelected().equals("SCD30"))
+    {
         Serial.println("-->[SLIB] SCD30 setting calibration to: " + String(ppmValue));
         scd30.setForcedRecalibrationFactor(ppmValue);
     }
-    if (getPmDeviceSelected().equals("CM1106")) {
+    if (getPmDeviceSelected().equals("CM1106"))
+    {
```

Comment:
Please for future PR not change lines that are not from the target of the PR. On the other hand, we try to use Google or clang C++ standard. More info:

https://stackoverflow.com/questions/45823734/visual-studio-code-formatting-for
https://zamhuang.medium.com/vscode-how-to-customize-c-s-coding-style-in-vscode-ad16d87e93bf